python/orca/test/bigdl/orca/ray/test_reinit_raycontext.py

```diff
@@ -54,9 +54,10 @@ def test_local(self):
             ray_ctx.stop()
             sc.stop()
             time.sleep(3)
-            for process_info in ray_ctx.ray_processesMonitor.process_infos:
-                for pid in process_info.pids:
-                    assert not psutil.pid_exists(pid)
+            # TODO: add the process monitor test in cluster mode
```

Comment:
for local mode, we have changed the implementation to directly call ray.init and thus there is no processmonitor.